README.md

```diff
@@ -251,6 +253,86 @@ capability might be supported but not enabled.
 | JSCVT     | Perform Conversion to Match Javascript
 | DCPOP     | Persistent Memory Support
 
+### Custom Features
+The Custom feature source allows the user to define features based on a mix of predefined rules.
+
+#### Rule Types
+
+##### PciId Rule
+The PciId rule allows matching the PCI devices in the system on the following PCI attributes: `class`,`vendor` and
+`device` identifiers.
+
+```yaml
+pciId :
+  class: ["<PCI_CLASS_ID>", ...]
+  vendor: ["<PCI_VENDOR_ID>", ...]
+  device: ["<PCI_DEVICE_ID>", ...]
```

Comment:
Possibly a personal preference, but, I would re-format this a bit, at least drop the quotes, for better readability.

I'd possibly also use lower case as a similar format is used with the other feature sources:
```yaml
  class: [<class id>, ...]
  vendor: [<vendor id>, ...]
  device: [<device id>, ...]
```

Reply:
will modify